config/routes.js

```diff
@@ -71,6 +71,11 @@ module.exports.routes = {
     action     : 'logout'
   },
 
+  '/auth/reddit/callback' : {
```

Comment:
As far as I can tell it wasn't defined -- /auth/reddit/callback was just calling the 'reddit' function in authController (the same as if /auth/reddit was called normally).